src/utils/__tests__/getFlowType-test.js

```diff
@@ -232,6 +232,39 @@ describe('getFlowType', () => {
     }, raw: '{ a: string, b: ?xyz }'});
   });
 
+  it('handles ObjectTypeSpreadProperty', () => {
+    var typePath = statement(`
+      var x: { ...OtherProps, ...MyType, a: string };
+
+      type MyType = { myString: string };
+    `).get('declarations', 0).get('id').get('typeAnnotation').get('typeAnnotation');
+
+    expect(getFlowType(typePath)).toEqual({
+      name: 'signature',
+      type: 'object',
+      signature: {
+        inheritedProperties: [
+          { name: 'OtherProps', value: { name: 'OtherProps' } },
+          { name: 'MyType',
```

Comment:
This should be inlined.

Reply:
That makes sense, because the ultimate goal of react-docgen is to describe the props of react components and not the complete AST of js files. So as long as we have all the information we should add them to properties I think.